### DISCUSSION TITLE
Voice message playback queue should not include own voice message

### DISCUSSION THREAD

**Discussion by grimhilt** [element-hq/element-meta#708] [Ideas]
#### What would you like to do?
When I start a queue of voice message, I would expect it to skip my own voice message and continue with the interlocutors voice messages.

#### Why would you like to do it?
Doing a voice message as listening to it is long and there are frequent use case where my voice message between other voice message I didn't listen to, and I do not need to listen to my own voice message, I just want to listen to the other one.

#### How would you like to achieve it?
Either adding an other settings (as this feature is useful when going through old conversation where listening to our own messages is needed for context).
Or just removing this feature.

**Comment by bwindels** [comment]
Hi, could you elaborate what you mean with `When I start a queue of voice message`. I am not aware of any kind of queue when playing back voice messages.

**Comment by bwindels** [comment]
Ah, I think I know what you mean now, if you play a voice message, it will try to play the next voice message.

**Comment by grimhilt** [comment]
Precisely, let's consider a conversation like this : 
* Voice Message 1 (Not me)
* Voice Message 2 (Not me)
* Voice Message 3 (Me)
* Voice Message 4 (Not me)

Currently, if I play 1, when it will reach the end, 2 will start, followed by 3 and 4.

I would expect my voice message to be ignore, so 1, 2, 4 will be played with the queue.
Or my voice message could have the same behavior of a text message and stop the queue, so when 1 is played, it will trigger 2 and that is it.

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:

**Comment by dbkr** [comment]
Conclusion from design (@gaelledel) is that including your own voice messages in the playback queue is intentional so you always get the whole context of the conversation (especially if it's something you're coming back to after some time).

**Comment by grimhilt** [comment]
Hi, I see your point. The issue I have is when I'm actively responding to voice messages by voice messages. In this case I have to manually skip my own messages to be able to listen to the next messages of my correspondent. And this tend to mess with the playback queue as my message is set to Paused Mode and when the next message finishes it will start again with my own paused message.

A potential solution could be to distinguish these two cases based on the time the message was sent compared to the current time.
